.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,13 @@
 **IMPORTANT: Please do not create a Pull Request without creating an issue first.**
 
-## Before submitting (checklist)
+## Before submitting (must do checklist)
 
-- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
-- [ ] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna-ts/blob/master/CONTRIBUTING.md)?
```

Comment:
it looked useful

Reply:
But then we ask for related issues. If it wasn't discussed there will be no related issues.
And for contribution guide I tend to agree)